src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,98 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+class Calculator {
+
+    private List<Character> separators;
```

Comment:
참조값이 변경되지 않는다면 불변화를 신경써줍시다 전반적으로 한번 신경써주세요!

```suggestion
    private final List<Character> separators;
```

Reply:
리스트 특성을 잠시 잊고 있었네요🥹 감사합니다!